cmd/metal-api/internal/datastore/image.go

```diff
@@ -35,3 +47,210 @@ func (rs *RethinkStore) DeleteImage(i *metal.Image) error {
 func (rs *RethinkStore) UpdateImage(oldImage *metal.Image, newImage *metal.Image) error {
 	return rs.updateEntity(rs.imageTable(), newImage, oldImage)
 }
+
+// MigrateMachineImages check images of all machine allocations and migrite them to semver images
+// must be executed only once.
```

Comment:
Wouldn't this be a typical task for a database migration?

Reply:
sure but how ?

Reply:
Mh... there are many projects that help with it. I already built his in at some point in time but decided it was too early to add it to the code base. But I think by now we should consider adding such a mechanism as otherwise we clutter the codebase with migration functions.